data-host/library/src/popup/components/SendTokens.tsx

```diff
@@ -68,11 +71,12 @@ function SpecifySend(props: {
   amount: bigint;
   onChangeAmount: (a: bigint) => void;
   onContinue: () => void;
+  onCancel: () => void;
 }) {
   const addressError = getAddressError(props.address);
   const validAmount = props.amount > BigInt(0);
   const isValid = addressError == null && validAmount;
-
+  const [loading] = useState(false);
```

Comment:
Why are you using a state with no setting? AFAIK, this is equivalent to just `false`.